frontend/packages/dev-console/src/components/topology/Topology.tsx

```diff
@@ -260,10 +296,13 @@ const Topology: React.FC<TopologyProps> = ({ data, serviceBinding, filters, acti
   );
 };
 
+const getServiceBindingStatus = ({ FLAGS }: RootState): boolean => FLAGS.get(ALLOW_SERVICE_BINDING);
```

Comment:
Last I reviewed (forgot to mention this - recalled when i saw this line) this exists already. Can you do a quick search for `ALLOW_SERVICE_BINDING` and see if it's implemented multiple times. Think we can probably reuse here.